beacon-chain/rpc/validator/server.go

```diff
@@ -125,32 +124,6 @@ func (vs *Server) ValidatorIndex(ctx context.Context, req *ethpb.ValidatorIndexR
 	return &ethpb.ValidatorIndexResponse{Index: index}, nil
 }
 
-// ExitedValidators queries validator statuses for a give list of validators
-// and returns a filtered list of validator keys that are exited.
-func (vs *Server) ExitedValidators(
```

Comment:
why are you removing this ?

Reply:
Unneeded endpoint, spoke with Preston offline.